tests/exercise_instructions.rs

```diff
@@ -348,6 +348,13 @@ fn fuzz_alu() {
             test_ins(true, format!("neg64 r{src}"), &mut prng, 21);
             test_ins(true, format!("neg32 r{src}"), &mut prng, 21);
 
+            test_ins(true, format!("le16 r{src}"), &mut prng, 21);
+            test_ins(true, format!("le32 r{src}"), &mut prng, 21);
+            test_ins(true, format!("le64 r{src}"), &mut prng, 21);
+            test_ins(true, format!("be16 r{src}"), &mut prng, 21);
+            test_ins(true, format!("be32 r{src}"), &mut prng, 21);
+            test_ins(true, format!("be64 r{src}"), &mut prng, 21);
```

Comment:
These are already tested on line 500, so now they are duplicates

Reply:
Ah, found them!
I'll remove the `be` ones and keep the `le` one. I'lld bring the close together too. Thanks for pointing that.